src/pages/news/2024-04-15-tbtc-is-for-l2s.md

```diff
@@ -1,6 +1,6 @@
 ---
 template: news-item
-title: "tBTC is for L2s: The Trust-Minimized Bitcoin Bridge for Layer 2 Scaling"
+title: "tBTC is for L2s: A Bitcoin Bridge Built for Scaling"
```

Comment:
“Built to Scale” maybe?

Also wondering about dropping the first header below as well.